### ISSUE TITLE
itemTextAlignment only works for single line MenuItems

### ISSUE THREAD

**Issue by StatusQuo** [danielsaidi/Sheeeeeeeeet#134]
![Simulator Screen Shot - iPhone 11 Pro Max - 2020-04-03 at 15 12 19](https://user-images.githubusercontent.com/735281/78364369-9d818000-75bd-11ea-89f2-0e0a2321eed5.png)

Added these to appeareance settings.
```
        item.itemTextAlignment = .center
        title.itemTextAlignment = .right
        item.titleColor = .red
```

Do you have any idea for the cause of the issue. I would like to help, but i don´t really know where to start.

**Comment by mohammadghk** [comment]
I tried to fix it here #115 , you can add it to your code manually as a temporary fix :)

**Comment by danielsaidi** [comment]
Thank you @mohammadghk - I dropped the ball on this, but I hope to be able to add better handling in a future version.

**Comment by danielsaidi** [comment]
Hi! I have been busy with various other projects, but finally got around to this.

I tried solving the text alignment for subtitle cells, but the problem is that the label frames are just as wide as the text, unlike regular cells where the label takes up the entire available width.

None of the things I tried helped solving this, but I found a bug in the code that made this worse. Instead of `.natural`, I have used `.left` as the standard text alignment, which has caused cells to render incorrectly for right to left locales.

I have fixed this bug in 3.1.3 and hope that this removes the most urgent need for this fix. Instead of setting text alignment, cells will now automatically adjust to the current locale.
